libgalois/include/katana/PropertyGraph.h

```diff
@@ -999,6 +998,8 @@ class KATANA_EXPORT EdgeTypeAwareTopology : public GraphTopologyTypes {
       const PropertyGraph* pg, const EdgeTypeIndex* edge_type_index);
 
   // TODO(amber): logic needs fixing for the case when there are nodes but no edges
+  // This will result in edge_type_index_->num_unique_types() == 0 and allocate
+  // adj_indices_ of size 0, even when graph has non-zero number of nodes.
```

Comment:
So `num_nodes` will return 0 even if there are nodes if there are no edges? If so that might be worth documenting specifically, even if this is temporary. It could be pretty surprising.

Reply:
Yes, this is the situation with a corner case. As I think about it, I have a better fix in mind. I'll push it in a bit. 
For my next PR, I'll pay attention to this aspect of corner cases, e.g. graph with 0 edges, or graph with 0 nodes and 0 edges.